mailpoet/tasks/phpstan/custom-stubs.php

```
@@ -100,15 +100,44 @@ public function get_end() {
+      public function get_meta( $key, $single = true ) {
+        return '';
+      }
+
+      /**
+       * Update meta data.
+       *
+       * @param string $key   Meta key.
+       * @param mixed  $value Meta value.
+       * @return void
+       */
+      public function update_meta_data( $key, $value ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$key'. (undefined)

(UnusedFormalParameter)